Ball.py

```diff
@@ -0,0 +1,15 @@
+# This file is for defining Ball class
+# Date: 2022.06.11
+
+from Color import Color
+import random
+
+class Ball:
+    def __init__(self):
+        self.color = self.setColor()
+
+    def setColor(self):
+        return random.choice([Color.RED, Color.GREEN, Color.BLUE, Color.YELLOW, Color.PURPLE])
```

Comment:
這裡怪怪的，通常會是 `def setColor(self, color)`
如果這裡只是初始值隨意的話，應該是
```suggestion
    def __init__(self):
        self.color = random.choice([Color.RED, Color.GREEN, Color.BLUE, Color.YELLOW, Color.PURPLE])
```

Reply:
順帶一提，把naming統一一下，method name要camelCase還是要snake_case呢？
我看了一下這個PR兩個都有用到